src/offerCallBuilder.js

```diff
@@ -0,0 +1,27 @@
+import {CallBuilder} from "./callBuilder";
+import {OrderbookCallBuilder} from "./orderbookCallBuilder";
+import {BadRequestError} from "./errors";
+let URI = require("URIjs");
+
+/**
+* @class OfferCallBuilder
+*/
+export class OfferCallBuilder extends CallBuilder {
+    /*
+    * @constructor
+    */
+    constructor(url, resource, ...resourceParams) {
+        super(url);
+        if (resource === 'accounts') {
+            this.url.segment([resource, ...resourceParams, 'offers']);
+        } 
+        // else if (resource === 'orderbook') {
```

Comment:
In general, I recommend that you only commit commented-out code alongside a note that explains when and how this block of code may be enabled in some definite future.

If the possibility of un-commenting this code is hazy, just go ahead and remove the block (which is the current situation for this code, IMO).

Random commented-out code is confusing for future developers... is this on purpose?  will it ever be uncommented?  Is it commented out because it's broken or because it's no longer needed?

Chances are, if you delete the block and need it in the future you'll recover it from git or you'll write it again from scratch except it will be better with your increase in experience over time.